recipe/build.sh

```diff
@@ -8,6 +8,7 @@ if [ `uname` == Darwin ]; then
     sed -i -e "s/@OSX_ARCH@/$ARCH/g" Lib/distutils/unixccompiler.py
     ./configure \
         --enable-ipv6 \
+        --enable-loadable-sqlite-extensions \
```

Comment:
@jjhelmus and @xmnlab we should probably factor out the common options to avoid that error in the future.

If @xmnlab is busy I can do that in another  PR.